jlab-sensing/ENTS-node-firmware
# Merge proto and calibration into single python module

**Name/Affiliation/Title**
John, UCSC, Maintainer

**Purpose of the PR**
- Integrate calibration code into published python package
- Refactor code into command line interface

**Development Environment**
`OS: Linux spruce 6.8.2-arch2-1 #1 SMP PREEMPT_DYNAMIC Thu, 28 Mar 2024 17:06:35 +0000 x86_64 GNU/Linux`
Hardware version: 2.2.1
Software version: branch
Platformio Version: 6.1.15

**Test Procedure**
Use `ents calib --help` to check command line options.
Run the calibration script with the necessary parameters.

**Additional Context**
- Did not set checks on voltage/current, instead currently defined as static values
- The entrypoint for calibration is messy and could use some refactoring.

**Task List**


**Relevant Issues**
Closes #124
